pkg/driver/controller_server.go

```diff
@@ -807,11 +806,6 @@ func (driver *Driver) controllerPublishVolume(
 		return nil, status.Error(codes.NotFound, err.Error())
 	}
 
-	if node.ChapUser != "" && node.ChapPassword != "" {
-		decodedChapPassword, _ := b64.StdEncoding.DecodeString(node.ChapPassword)
```

Comment:
Hey @suneeth51 I was just reviewing this with @sneharai4 and we realized that we need to return unencrypted password to CSP.

Reply:
Sure, I will fix that.